src/python/grpcio/grpc/_server.py

```diff
@@ -591,7 +591,13 @@ def _handle_call(rpc_event, generic_handlers, thread_pool,
     if not rpc_event.success:
         return None, None
     if rpc_event.request_call_details.method is not None:
-        method_handler = _find_method_handler(rpc_event, generic_handlers)
+        try:
+            method_handler = _find_method_handler(rpc_event, generic_handlers)
+        except Exception as e:  # pylint: disable=broad-except
+            details = 'Exception servicing handler: {}'.format(e)
+            logging.exception(details)
```

Comment:
Probably log the full stack trace instead of the details

(I realize we are logging the details if the invocation fails too, but I think the stack trace is more useful there as well)

Reply:
I am ambivalent about this decision, but if we want to do it, we should do it in another PR consistently everywhere.

Reply:
Isn't the whole point of `logging.exception` that it logs the stack trace for you? @kpayson64: are you saying that the code in its current form does not log the full stack trace?

Reply:
Looks like it does log stack traces.

Reply:
Looks like it.  I had assumed that string-formatting the exception wouldn't output the full stack trace.